app/actions/CompanyInterestActions.js

```diff
@@ -83,3 +83,29 @@ export function updateCompanyInterest(
     ).then(() => dispatch(addToast({ message: 'Bedriftsinteresse endret!' })));
   };
 }
+
+export function fetch({
+  next,
+  filters
+}: { next: boolean, filters: Object } = {}): Thunk<*> {
+  return (dispatch, getState) => {
+    const cursor = next ? getState().companyInterest.pagination.next : {};
+
+    return dispatch(
+      callAPI({
+        types: CompanyInterestForm.FETCH_MORE,
```

Comment:
In order to get the `fetching`, `hasMore` and the `next` cursor to work, the type of the action must be the one                           declared as the `fetchType` when creating the entityReducer. https://github.com/webkom/lego-webapp/blob/86f6e649ed68e8b7d15e18b7a2b2a1ca8d870f4e/app/reducers/companyInterest.js#L24

Switching to `CompanyInterestForm.FETCH_ALL` will solve the problem :smile: